### PR TITLE
Update jCrop from V0.9.12 to V0.9.15

### PR DESCRIPTION
**Pull request by ayumihamsaki2**
Have updated jCrop from V0.9.12 to V0.9.15 I have also included `jquery.color` as this version needs this now. Did a quick search in this repo and couldn't find this new file, so have included it in the folder.

Ref: https://github.com/octobercms/october/issues/3867

### CODE REVIEW

File: modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js [hunk 1]
```diff
@@ -1084,10 +1081,9 @@
       //}}}
 
       // This is a hack for iOS5 to support drag/move touch functionality
-      // Hack OctoberCMS - the event handler was moved to the Touch module.
-      // The closure used before was handling a reference to the target object,
-      // preventing it from removing from DOM after the control is destroyed.
-      $(document).bind('touchstart.jcrop-ios', Touch.fixTouchSupport);
```
Comment (ayumihamsaki) [inline modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js]: Need to discuss this line

Comment (w20k) [reply modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js]: I think we should revert this part or find why it was introduced as a `fix` :)

**Comment by ayumihamsaki** [comment]
Don't understand why this is failing ?

**Comment by w20k** [comment]
Don't worry about that for now :) Old/new test issue.

File: modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js (hunk 1 above)
Comment (ayumihamsaki) [reply modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js]: Agree

Comment (ayumihamsaki) [reply modules/backend/assets/vendor/jcrop/js/jquery.Jcrop.js]: Added commit: https://github.com/octobercms/october/pull/4281/commits/448f543ac8236aaec1d04f68b1d8cdeb045b6fd8

**Comment by LukeTowers** [comment]
@ayumihamsaki 
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/.github/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

**Comment by ayumihamsaki** [comment]
Sorry I usually check to make sure it goes to the dev branch.

**Comment by ayumihamsaki** [comment]
Sorry I usually check to make sure it goes to the dev branch.

**Comment by LukeTowers** [comment]
@ayumihamsaki you'll need to push another commit to trigger travis again, tests broke because it was originally submitted to the master branch.

**Comment by ayumihamsaki2** [comment]
I'll create a fresh pr - github seems all messed up today.

**Comment by ayumihamsaki** [comment]
See https://github.com/octobercms/october/pull/4292 for new pr

**Pull request closed without merging**